pinot-connectors/pinot-flink-connector/src/main/java/org/apache/pinot/connector/flink/sink/PinotSinkFunction.java

```diff
@@ -49,6 +50,8 @@ public class PinotSinkFunction<T> extends RichSinkFunction<T> implements Checkpo
   public static final int DEFAULT_EXECUTOR_POOL_SIZE = 5;
   public static final long DEFAULT_EXECUTOR_SHUTDOWN_WAIT_MS = 3000;
 
+  public static final String DEFAULT_UPLOADED_REALTIME_SEGMENT_PREFIX = "flink";
```

Comment:
should we keep the default prefix as "uploaded_flink" for more clarity?

Reply:
No prefernece here, the segment naming convention implies uploaded realtime segment, so putting uploaded in prefix seems redundant. Let me know if you see any issues from minion operations perspective.